next-env.d.ts

```diff
@@ -1,3 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
```

Comment:
What is purpose of this comment here.

Reply:
I have no idea how it got there..

Reply:
Don't delete this file, it was already there. Just don't make the changes

Reply:
That's a Next js generated file.
It might get updated on `yarn install`.

Run `yarn install` and push the updated version of the file.

Also, you should check what files you are adding to your commits.

> I have no idea how it got there..

☝🏽This is not a good answer